.github/workflows/interface-unit-tests.yml

```diff
@@ -365,12 +368,12 @@ jobs:
       # This is required during the release process when the latest released version of
       # catalyst requires the latest version of pennylane that is about to be released.
       # Installing catalyst after pennylane to make sure that the latest catalyst is used.
-      install_catalyst_after_pennylane: true
+      install_catalyst_nightly: true
       # using lightning master does not work for the tests with external libraries
       install_pennylane_lightning_master: false
       pytest_coverage_flags: ${{ inputs.pytest_coverage_flags }}
       pytest_markers: external
-      additional_pip_packages: pyzx pennylane-catalyst matplotlib stim quimb
+      additional_pip_packages: pyzx matplotlib stim quimb mitiq pennylane-qiskit ply
```

Comment:
All three dependencies have been added for the mitiq tests. CI was failing without `ply` (see [here](https://github.com/PennyLaneAI/pennylane/actions/runs/10708074255/job/29689553521?pr=6214#step:8:665))

Reply:
This has been moved to a different PR #6216